src/lib.rs

```diff
@@ -152,6 +152,7 @@ pub use auxinfo::{
     info::{AuxInfoPrivate, AuxInfoPublic},
     participant::AuxInfoParticipant,
 };
+// TODO export status type.
```

Comment:
nit: remove outdated docs (I think you handled this?)

Reply:
Did you mean to leave this in? If so, could you reference #314, or whichever issue you think will address it?

Reply:
Whoops. I thought I had addressed it, thanks!